src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public record FruitTransaction(Operation operation,
+                               String fruit, int quantity) {
```

Comment:
add fields of transaction itself - fruit, quantity, constructor, getters setters etc